engine/events/map_name_sign.asm

```diff
@@ -276,7 +280,18 @@ LoadMapNameSignGFX:
 	rst AddNTimes ; preserves bc
 	ld de, wBGPals1 palette PAL_BG_TEXT
 	call FarCopyColorWRAM
-	jmp SetDefaultBGPAndOBP
+
+	; copy sign palette for PAL_BG_TEXT
+	ld hl, SignPals
+	ld bc, 1 palettes
+	ld a, [wSign]
+	rst AddNTimes ; preserves bc
+	ld de, wBGPals2 palette PAL_BG_TEXT
+	call FarCopyColorWRAM
+
+	ld a, TRUE
+	ldh [hCGBPalUpdate], a
```

Comment:
Is this an equivalent alternative to the `jmp SetDefaultBGPAndOBP` that was here before? Or if not, what's the difference?

Reply:
No, `SetDefaultBGPAndOBP` will copy all pals from `wBGPals1` / `wOBPals1` to `wBGPals2` / `wOBPals2` and then request a pal update through `hCGGPalUpdate` (Used to tell vblank to copy palls to `rBGPD`)...  We don't want to do that anymore, as this would instantly cause our map connection fading to end prematurely.